.github/workflows/ci.yml

```diff
@@ -23,19 +23,26 @@ jobs:
     - name: Checkout security
       uses: actions/checkout@v2
 
+    - name: Cache Maven packages
+      uses: actions/cache@v1
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+
     - name: Checkstyle
-      run: mvn checkstyle:checkstyle
+      run: mvn -B checkstyle:checkstyle
 
     - name: Test
-      run: OPENDISTRO_SECURITY_TEST_OPENSSL_OPT=true mvn test
+      run: OPENDISTRO_SECURITY_TEST_OPENSSL_OPT=true mvn -B test
```

Comment:
Out of curiosity will batch mode reduce our test run time?

Reply:
No, batch mode won't reduce the test time, but it should route all output to Github captured logs.